commands/arm/parseBVH.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh 
+# takes in two arguments, 1 which is the BVH file and a second for the output, converts only the rotation on each linkage that we want
+tail -n +38 "$1" | awk '{ print $4,$7,$10; }' > "$2"
```

Comment:
I can't read this at all so I'm going to assume it's fine

Reply:
cuts off the first 38 lines of the BVH file, and then takes the 4th, 7th, and 10th things in each line and puts it into the file
shell scripting magic! look up: awk